tock/tock/static/sass/_entries.scss

```diff
@@ -3,12 +3,13 @@
 		border-bottom: 2px solid $theme-color-base-light;
 	}
 	select {
-		width:100%;
+		width: 65%;
 		color: $theme-color-base-dark;
-		background-color: #fff;
+		background-color:transparent;
 		border: 1px solid $theme-color-base-dark;
 		transition: border-color ease-in-out .15s,box-shadow ease-in-out .15s;
 		margin-bottom:0;
+    font-size: 19px;
```

Comment:
**question:** An explicit font size in pixels seems like a strange thing to add here. What problem was this meant to solve and is there a more idiomatic way to fix it in the US Web Design System?

Reply:
i put these information based on the other field's set (input of hours)